src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx

```diff
@@ -624,12 +623,12 @@ For users on our New Relic One user model, a "group" is what allows the grouping
 
 We have two default groups:
 
-* **User**: has the [**All product admin** role](#standard-roles), which grants product feature-related access and configuration abilities but not the organization-level admin abilities (like the ability to manage users or billing).
+* **User**: This group allows a user to use and configure monitoring/analysis features but **not** perform account-related tasks like managing billing or users. It has access to the **All product admin** role, which gives access to our observability platform tools but not to the organization and user management capabilities governed by the **Organization manager** and **Authentication manager** roles. 
 * **Admin**: has full access and capabilities, including the organization-level admin abilities. This is the equivalent of having the **All product admin**, the **Billing user**, the **Organization manager** and the **Authentication domain manager** [roles](#standard-roles). 
 
 These groups are added inside your default [authentication domain](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/), which includes the default settings of users a) being managed via New Relic and b) logging in via standard email and password. If you add other authentication domains (for SAML SSO and/or SCIM provisioning of users), you'd have new custom groups in those new domains to govern those users. 
 
-Note that groups, whether default or custom, are not what give a user their permissions; the role or roles granted to that group are what does that. If your organization has multiple accounts and want to better understand how access grants give user groups access to roles and accounts, see [Access grants](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
+Note that groups, whether default or custom, are not what give a user their capabilities; the role or roles granted to that group are what grants a user their capabilities (with [basic user](#user-type) restrictions overriding that). If your organization has multiple accounts and want to better understand how access grants give user groups access to roles and accounts, see [Access grants](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
```

Comment:
If your organization has multiple accounts and want**S**.  

This sentence is a little confusing though. Could we break it in two?